postBackend.py

```diff
@@ -55,6 +36,29 @@ def get_posts():
         resp = jsonify(newPost), 201
         return resp
 
+@app.route('/filter', methods=['POST'])
```

Comment:
No comments and looks good. The way you are accepting data from the front end should match how I am sending you the data. This code is clean :)